doc/rados/troubleshooting/troubleshooting-pg.rst

```diff
@@ -349,77 +349,106 @@ may confuse applications that expect the object to exist.
 Homeless Placement Groups
 =========================
 
-It is possible for all OSDs that had copies of a given placement groups to fail.
-If that's the case, that subset of the object store is unavailable, and the
-monitor will receive no status updates for those placement groups.  To detect
-this situation, the monitor marks any placement group whose primary OSD has
-failed as ``stale``.  For example::
+It is possible that ever OSD that had copies of a given placement group could
```

Comment:
Every

Reply:
Suggest recasting the paragraph in present tense, can fail , if this happens …

Reply:
Good catch, and I accept the suggest to change the paragraph's tense.

Accepted.